src/dmn/trn_agent_xdp_usr.h

```diff
@@ -131,6 +133,16 @@ int trn_agent_get_endpoint(struct agent_user_metadata_t *umd,
 int trn_agent_delete_endpoint(struct agent_user_metadata_t *umd,
 			      struct endpoint_key_t *epkey);
 
+int trn_agent_update_packet_metadata(struct agent_user_metadata_t *umd,
+			      struct packet_metadata_key_t *key,
+			      struct packet_metadata_t *packet_metadata);
+
+int trn_agent_get_packet_metadata(struct agent_user_metadata_t *umd,
+			   struct packet_metadata_key_t *key, struct packet_metadata_t *packet_metadata);
```

Comment:
Let's stick with C coding conventions (80 cols). struct packet_metadata_t goes in the next line for consistency with the rest of the code.

Reply:
Fixed.